opendcs-rest-api/src/main/java/org/opendcs/odcsapi/dao/DbException.java

```diff
@@ -18,28 +18,28 @@
 public class DbException extends Exception
 {
 	private static final long serialVersionUID = 4552935582551955795L;
-	private String module = null;
-	private Exception cause = null;
-	
-	public DbException(String msg)
-	{
-		super(msg);
-	}
+	private final String module;
```

Comment:
If this is because of the existing things go ahead and just ditch it. We was used for identifying things in the logs but we're moving to slf4j with the full class names over time.

Reply:
yeah, just too much code to update in this PR to completely ditch it, that's why I endeded up overloading. Better to just cut over to a new class though.